config/jobs/kubernetes-sigs/sig-windows/release-master-windows-presubmits.yaml

```
@@ -405,6 +406,18 @@ presubmits:
           env:
           - name: GINKGO_FOCUS
             value: \[sig-windows\] # run just a subset to speed up testing time
+          volumeMounts:
+          - mountPath: /var/run/secrets/azure-token/serviceaccount
+            name: azure-token
+            readOnly: true
+      volumes:
+        - name: azure-token
+          projected:
+            defaultMode: 420
+            sources:
+              - serviceAccountToken:
+                  expirationSeconds: 86400
+                  path: token
```

```suggestion
                  path: token
                  audience: api://AzureADTokenExchange
```
You can configure the audience to the recommended value for AZWI: `api://AzureADTokenExchange`
(xref: https://github.com/Azure/azure-workload-identity/blob/main/pkg/webhook/consts.go#L58)